src/verifiers/account_verifier.cairo

```diff
@@ -150,22 +124,13 @@ func verify_account{
         pow2_array=pow2_array,
     );
 
-    // write verified account state
-    assert account_values[account_value_idx] = AccountValues(values=value, values_len=value_len);
-
     // add account to memorizer
     AccountMemorizer.add(
         chain_id=chain_id,
         block_number=account_proof.block_number,
         address=account.address,
-        index=account_value_idx,
+        rlp=value,
```

Comment:
We are dropping value_len information, how do we later know amount of felts to read?

Reply:
ah it is rlp_list_retrieve in decoder isnt it?

Reply:
yup, we now get the lengths from the actual rlp decoding.